data/os/RedHat.yaml

```diff
@@ -0,0 +1,14 @@
+---
+profile_system_auth::config::removed_pkgs:
+  - "fprintd-pam"
+  - "authconfig"
+profile_system_auth::config::required_pkgs:
+  - "authselect"
+  - "oddjob"
+  - "oddjob-mkhomedir"
+profile_system_auth::config::use_authconfig: false
+profile_system_auth::kerberos::required_pkgs:
+  - "krb5-libs"
+  - "krb5-workstation"
+profile_system_auth::ldap::required_pkgs:
+  - "openldap-clients"
```

Comment:
I have now moved all of the RedHat specific parameters out of the common.yaml file and into the ./os/RedHat.yaml file.